scripts/mock_data/supplementary_data_no_repeat.json

```diff
@@ -1,8 +1,9 @@
 {
   "dataset_id": "c067f6de-6d64-42b1-8b02-431a3486c178",
   "survey_id": "123",
+  "encryption_key_id": "df88fdad2612ae1e80571120e6c6371f55896696",
```

Comment:
To be removed.

Reply:
How come this isn't needed?

Reply:
The encryption key id is encoded within the JWE token header, so can be decoded and used to select the key to use. Mebin is in the process of getting this change reflected on the SDS side.